lib/features/cleanup/domain/model/recent_login_url_cleanup_rule.dart

```diff
@@ -0,0 +1,11 @@
+
+import 'package:tmail_ui_user/features/cleanup/domain/model/cleanup_rule.dart';
+
+class RecentLoginUrlCleanupRule extends CleanupRule {
+  final int storageLimit;
+
+  RecentLoginUrlCleanupRule({this.storageLimit = 10}) : super();
```

Comment:
IMO, we should keep this in db, no need to clean up it in someday

Reply:
okie, keep up to 10 is ok to me

Reply:
If the user uses it more than once, login on multiple servers. IMO, in the long run we also need it to clean up memory to avoid wasting machine resources.